src/runtime/audience-action-flow.ts

```diff
@@ -321,6 +330,69 @@ export class AudienceActionFlow {
   }
 
   /**
+   * Populates localStorage with PPS configuration parameters based on
+   * SurveyDataTransferRequest.
+   * @param {SurveyDataTransferRequest} request
+   * @return {boolean}
+   * @private
```

Comment:
Since this is TypeScript, types don't need to be specified in JSDoc. The `request` param can be removed from the doc (the type is in the param declaration). The return type and visibility (private) should be specified on the method itself.

Reply:
sorry about that! the migration happened while i was still in the process of writing this PR which is why i had the annotations.